lnrpc/neutrinorpc/config_active.go

```diff
@@ -0,0 +1,34 @@
+//go:build neutrinorpc
+// +build neutrinorpc
+
+package neutrinorpc
+
+import (
+	sync "sync"
+
+	"github.com/lightninglabs/neutrino"
+)
+
+// Config is the primary configuration struct for the neutrino RPC server. It
+// contains all the items required for the rpc server to carry out its
+// duties. The fields with struct tags are meant to be parsed as normal
+// configuration options, while if able to be populated, the latter fields MUST
+// also be specified.
+type Config struct {
+	// curFilteredBlockHeight lets the rescan goroutine track where it
+	// thinks the chain is based on OnFilteredBlockConnected and
+	// OnFilteredBlockDisconnected.
+	CurFilteredBlockHeight int32
+
+	// ChainService is required to handle neutrino chain service requests.
+	NeutrinoCS *neutrino.ChainService
+
+	rescanRunning bool
```

Comment:
Given this is just a config, I don't think the fields below are relevant, as they can live closer to the actual sub-server implementation itself.